### PR TITLE
14794 - Added Managing Your Connection e2e.

### LINKED ISSUE #14794: Create E2E tests for the Managing Your Connection Page
## Problem statement
This page on the RS website needs End-to-end (E2E) tests created to ensure that it works properly before merging into master. Without E2E tests, some features/links may not function/load properly and we wouldn't know until a user tried to use them.

The Page is located at: https://reportstream.cdc.gov/managing-your-connection

## What you need to know
- This is based off the investigation from #11307
- [E2E Test Creation Tracker](https://cdc.sharepoint.com/:x:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7B6EB440E8-4A08-4F4F-A2C1-ED7BB23E665B%7D&file=E2E%20Test%20Tracker.xlsx&action=default&mobileredirect=true&wdsle=0)
- Strategy discussed for checking content load: check to see if every DIV on this page loaded

## Acceptance Criteria
As any user, logged in or not, check that:
- [x] The page loads
- [x] If there is an error, the error is shown on the page
- [x] Page content loads without errors (including if the footer is showing, but not its functionality) (also, not checking the content itself, but just if it loads)
- [x] Components work as expected (eg, accordion expands/collapses, side nav, etc)

### PR DESCRIPTION
**Pull request by penny-lischer**
This PR ...

Added Managing Your Connection e2e and updates to other e2e spec files.

Test Steps:
1. Run e2e tests locally `yarn run test:e2e-ui`

## Changes
Screenshot 2024-07-18 at 12 14 20 PM

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

## Linked Issues
- Fixes #14794

### CODE REVIEW

File: frontend-react/e2e/spec/all/managing-your-connection-page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        });
+        await page.goto();
+        await use(page);
+    },
+});
+
+test.describe(
+    "Managing Your Connection page",
+    {
+        tag: "@smoke",
```
Comment (etanb) [inline frontend-react/e2e/spec/all/managing-your-connection-page.spec.ts:62]: what factors are we using to determine what should be a smoke test or not?

File: frontend-react/e2e/spec/all/managing-your-connection-page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        test("has correct title", async ({ managingYourConnectionPage }) => {
+            await expect(managingYourConnectionPage.page).toHaveTitle(
+                managingYourConnectionPage.title,
+            );
+            await expect(managingYourConnectionPage.heading).toBeVisible();
+        });
+
+        test.describe("Quick links", () => {
+            for (const card of cards) {
+                // eslint-disable-next-line playwright/expect-expect
```
Comment (etanb) [inline frontend-react/e2e/spec/all/managing-your-connection-page.spec.ts:74]: We've implicitly agreed that we're ok with conditional tests, ie test within loops, we should probably just update our eslint config?

**Review by etanb — APPROVED** [review]
LGTM w/some comments

File: frontend-react/e2e/spec/all/managing-your-connection-page.spec.ts (hunk 1 above)
Comment (penny-lischer) [reply frontend-react/e2e/spec/all/managing-your-connection-page.spec.ts:62]: We all(devs) decided as a team on 7/3/2024 that we would test all `happy path scenarios` in public facing pages and all `happy path scenarios` for admins in auth pages,

Comment (etanb) [reply frontend-react/e2e/spec/all/managing-your-connection-page.spec.ts:62]: awesome! might be worth adding that to the README for any future public facing pages + devs.

**Review by jpandersen87 — APPROVED** [review]

**Pull request merged**